src/publisher/middleware.py

```diff
@@ -143,12 +56,12 @@ def requested_version(request, response):
     return (response_mime[0], "*" if not versions else max(versions))
 
 
-def deprecated(request):
-    if not settings.API_V12_TRANSFORMS:
-        return False
+def is_deprecated(request):
     targets = [
         ("application/vnd.elife.article-poa+json", "version", b"1"),
         ("application/vnd.elife.article-vor+json", "version", b"1"),
+        # 2020-06
+        ("application/vnd.elife.article-vor+json", "version", b"2"),
```

Comment:
can we say the oldest in the group is deprecated? in which case, see settings/ALL_SCHEMA_IDX